1-mandatory/2-people-i-know.js

```diff
@@ -443,19 +443,23 @@ This time, I only want the ----> full names of the people who can multitask
 //     -> colleagues array of objects
 //        -> skills array
 
-/////////////////// HELP!! /////////////////////////
-
-let multitaskingColleagues = friends.filter((singleFriend) =>
-  singleFriend.colleagues
-    .map((colleague) => colleague.skills) // how do I access and search inside the skills array?? Do I add another .map?
-    .includes("Multi-tasking")
-);
-
-let colleaguesWhoCanMultitask = multitaskingColleagues.map(
-  (singleFriend) => singleFriend.name.first + " " + singleFriend.name.last
+let colleaguesWhoCanMultitask = [];
+let allColleaguesArrayOfArrays = friends.map(
+  // here we are mapping the colleagues to the outer friends array as we don't need all the other stuff
+  (eachFriend) => eachFriend.colleagues
 );
```

Comment:
I found another way easier than this `let colleaguesWhoCanMultitask = [];
for (const friend of friends) {
  for (const colleague of friend.colleagues) {
    for (const skill of colleague.skills) {
      if (skill === "Multi-tasking") {
        colleaguesWhoCanMultitask.push(colleague.name)
      }
    }
  }
}`